cmd/filebrowser/main.go

```diff
@@ -2,25 +2,22 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net"
-	"net/http"
-	"os"
-	"path/filepath"
-	"strings"
-
 	"github.com/asdine/storm"
-
-	"gopkg.in/natefinch/lumberjack.v2"
-
 	"github.com/filebrowser/filebrowser"
 	"github.com/filebrowser/filebrowser/bolt"
 	h "github.com/filebrowser/filebrowser/http"
 	"github.com/filebrowser/filebrowser/staticgen"
 	"github.com/hacdias/fileutils"
 	flag "github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	"gopkg.in/natefinch/lumberjack.v2"
+	"io/ioutil"
+	"log"
+	"net"
```

Comment:
I wonder why all of these imports moved here. 

By the way, @1138-4EB just found out CircleCI had the option to build forked PRs disabled. I just enabled it, but I believe we'll need changes on this PR to trigger it.

Reply:
They are sorted in alphabetical order now. I think this is because `gofmt` was used. Should we use `goimports` instead? See http://goinbigdata.com/goimports-vs-gofmt/